src/encoded/commands/extract_test_data.py

```diff
@@ -138,15 +141,13 @@ def main():
         help="directory to write filtered tsv files to.")
     args = parser.parse_args()
     pipeline = anon_pipeline() if args.anonymize else extract_pipeline()
-    import pdb
-    import sys
-    import traceback
     try:
         run(pipeline, args.inpath, args.outpath)
-    except:
+    except Exception:
         type, value, tb = sys.exc_info()
         traceback.print_exc()
-        pdb.post_mortem(tb)
+        import pdb; pdb.post_mortem(tb)
```

Comment:
out of curiosity, why import pdb only on Exception?

Reply:
As Eric points out this is probably for debugging and should be removed.

Reply:
I'm very happy to remove it. I wasn't sure if it was responsible for obtaining some important debugging info. But I hate seeing `pdb` imported at all. I usually imagine it the programming equivalent of a surgeon closing up a patient with his scalpel still inside.

Reply:
Whoops. I made this edit in a cgap repo I had open at the same time. That means it didn't make it into the PR. But this has been here a long time and can wait until another day. I'll definitely remove it on a future PR. I had been meaning to ask if anyone minded, and am happy to hear no one does. :)